modules/bpi/bpi.push.inc

```diff
@@ -222,6 +222,15 @@ function bpi_convert_to_bpi($node, $category, $audience, $with_images = FALSE, $
     );
   }
 
+  $tags = array();
+  $tags_field = field_view_field('node', $node, variable_get('bpi_field_tags', ''));
```

Comment:
Since we do not retrieve the tags with the intent of displaying them then how about using [`field_get_items()`](https://api.drupal.org/api/drupal/modules%21field%21field.module/function/field_get_items/7) instead of `field_view_field()`?

I can see that `field_view_field()` is being used elsewhere in the code but I see no reason for doing so there either and changing it is of out scope of this review.

Also: How well does this code handle `bpi_field_tags` not being set?